Author: Eric Desrochers <eric.desrochers@canonical.com>
Description: mdcheck fix empty spaces in timer unit files (LP: #1852747).
 dh_installsystemd is failling thus FBTFS because of the spaces
 in the systemd unit timer files directive.

Index: mdadm-devuan/systemd/mdcheck_continue.timer
===================================================================
--- mdadm-devuan.orig/systemd/mdcheck_continue.timer
+++ mdadm-devuan/systemd/mdcheck_continue.timer
@@ -14,4 +14,4 @@ RandomizedDelaySec=12h
 Persistent=true
 
 [Install]
-WantedBy= mdmonitor.service
+WantedBy=mdmonitor.service
Index: mdadm-devuan/systemd/mdcheck_start.timer
===================================================================
--- mdadm-devuan.orig/systemd/mdcheck_start.timer
+++ mdadm-devuan/systemd/mdcheck_start.timer
@@ -14,5 +14,5 @@ RandomizedDelaySec=24h
 Persistent=true
 
 [Install]
-WantedBy= mdmonitor.service
-Also= mdcheck_continue.timer
+WantedBy=mdmonitor.service
+Also=mdcheck_continue.timer
Index: mdadm-devuan/systemd/mdmonitor-oneshot.timer
===================================================================
--- mdadm-devuan.orig/systemd/mdmonitor-oneshot.timer
+++ mdadm-devuan/systemd/mdmonitor-oneshot.timer
@@ -14,4 +14,4 @@ RandomizedDelaySec=24h
 Persistent=true
 
 [Install]
-WantedBy= mdmonitor.service
+WantedBy=mdmonitor.service
